pkg/meta/meta.go

```diff
@@ -122,7 +122,7 @@ func (env *Environment) UpdateComponents(specs []string, nightly, force bool) er
 			component, v := ParseCompVersion(spec)
 			v1specs = append(v1specs, repository.ComponentSpec{ID: component, Version: v.String(), Force: force})
 		}
-		return env.v1Repo.UpdateComponents(v1specs)
+		return env.v1Repo.UpdateComponents(v1specs, nightly)
```

Comment:
I think nightly should be added to the component specs, not passed as another argument.